src/main/resources/data/malum/tags/items/spirit_crystals.json

```diff
@@ -0,0 +1,12 @@
+{
+  "values": [
+    "malum:sacred_spirit",
+    "malum:wicked_spirit",
+    "malum:arcane_spirit",
+    "malum:eldritch_spirit",
+    "malum:aerial_spirit",
+    "malum:aqueous_spirit",
+    "malum:earthen_spirit",
+    "malum:infernal_spirit"
```

Comment:
Adding malum items to a malum tag? o.O

Reply:
Yeah uhh.. i was kinda tired when I wrote this, this should probably not be in malum's namespace, but it turns out that malum just.. doesn't **have** a tag for spirit crystals!